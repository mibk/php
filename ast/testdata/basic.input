<?php   namespace \Dubax \
Foo;use\
DateTime  ;use\This
\Awesome\  Use\Statement;const
AA = 'tuzkovy'. 'baterky'
;/**@var    string*/$bb    =
  'auto'
;$cc

;class// ignore
Bar {function cleanup(  $foo,  $bar ,
){ echo  'something'
;  if (/* don't fmt just yet*/ )
{
	$this ->{ $foo
}	;switch;
lookup();$x->
         class( 'y' )
;	}


}function xx(){} }function toplevel ($n
  ){return 0     ;}

function xx1 (  ) :int { return  1 ;}