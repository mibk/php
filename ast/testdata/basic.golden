<?php

namespace Dubax\Foo;

use DateTime;
use This\Awesome\Use\Statement;

const AA = 'tuzkovy'. 'baterky';

class Bar
{
	function cleanup($foo, $bar) {
		echo  'something';
		if (/* don't fmt just yet*/ ){
			switch;
		}
;
	}

	function xx() {
	}
}
